src/promptflow/promptflow/_sdk/entities/_run.py

```diff
@@ -348,6 +348,10 @@ def _to_dict(self):
             # add exception part if any
             exception_dict = local_storage.load_exception()
             if exception_dict:
+                if exclude_additional_info:
```

Comment:
Nit: set to `_error` so LOCAL and RUN_HISTORY can share exception handle logic.

Reply:
good catch, let me test this, ideally that's good

Reply:
I think no, LOCAL and RH has little different structure, RH has one extra level, so we need to operate on `errors.errors`, while LOCAL only need to handle `errors`.